src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -2275,4 +2275,23 @@ limitations under the License.
             </include>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>Cooktop</name>
+        <domain>CHIP</domain>
+        <typeName>Matter Cooktop</typeName>
+        <class>Simple</class>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x0078</deviceId>
+        <clusters lockOthers="true">
+            <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true">
+                <requireAttribute>DEVICE_TYPE_LIST</requireAttribute>
+                <requireAttribute>SERVER_LIST</requireAttribute>
+                <requireAttribute>CLIENT_LIST</requireAttribute>
+                <requireAttribute>PARTS_LIST</requireAttribute>
```

Comment:
Why are these here?  They don't need to be here; those are always mandatory attributes on Descriptor, no?

And yes, I know some other (non-spec, note) device types in this file do this.  Those are wrong.

Reply:
Thanks. Done. Pushed.